ya-runtime-erigon/Cargo.toml

```diff
@@ -24,4 +24,4 @@ titlecase = "1.1.0"
 tokio = { version = "0.2", features = ["io-std", "io-util", "rt-core", "rt-threaded", "macros", "process", "time"] }
 
 [patch.crates-io]
-ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "880efaf19111e71647dea23fd997ebf6edc6e3d7" }
+ya-runtime-sdk = { git = "https://github.com/golemfactory/ya-runtime-sdk.git", rev = "5d857c5e2fccb5b28a89b5b6fcf78d4c34649c5f" }
```

Comment:
@mfranciszkiewicz Maybe we can have a release (tagged version)?

Reply:
@pnowosie @Wiezzel I've released `ya-runtime-sdk` to crates.io

Reply:
Good catch! It's time to switch to crates.

Reply:
BTW @mfranciszkiewicz @Wiezzel switching to crates.io breaks the build (now)
```
    Updating crates.io index
error: failed to resolve patches for `https://github.com/rust-lang/crates.io-index`

Caused by:
  patch for `ya-runtime-sdk` in `https://github.com/rust-lang/crates.io-index` points to the same source, but patches must point to different sources
Command \'cargo build\' has failed!
```
(checked on mac & ubuntu)